digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteIntegrationApi.scala

```diff
@@ -171,8 +202,8 @@ class ViiteIntegrationApi(val roadAddressService: RoadAddressService, val roadNa
         BadRequest(message)
       } else {
         try {
-          val changesSince = DateTime.parse(muutospvm.get, dateFormat)
```

Comment:
Does this work still with the old Tierekisteri implementation?

Reply:
Yes, I tested with local TR  connecting to local VIITE.